### ISSUE TITLE
Color theme buttons have no accessible name

### ISSUE THREAD

**Issue by dankeck** [Hiebeler/socialfolio-frontend#15]
The buttons to set light and dark color themes, as well as the X button to close the menu in mobile view, have no accessible name. Using a screen reader, I only heard the word "button" and would not have known what the button is for.

![Mobile view of Socialfolio with the menu open. There are four icons: a laptop, a sun, a moon, and a letter X to close the menu.](https://github.com/user-attachments/assets/b3955538-91bc-4de0-b438-b90e68ad38af)

**Comment by Hiebeler** [comment]
Thanks for the issue, I added accessible names to all Icon buttons.
